unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesAndEventsCardsFactory.cs

```diff
@@ -74,28 +74,54 @@ public static Pool GetCardsPoolLazy(string poolName, BaseComponentView cardPrefa
         return modal;
     }
 
-    public static List<PlaceCardComponentModel> ConvertPlaceResponseToModel(List<PlaceInfo> placeInfo)
+    public static List<PlaceCardComponentModel> ConvertPlaceResponseToModel(IEnumerable<PlaceInfo> placeInfo, int amountToTake)
     {
         List<PlaceCardComponentModel> modelsList = new List<PlaceCardComponentModel>();
+        int count = 0;
+        foreach (var place in placeInfo)
+        {
+            modelsList.Add(
```

Comment:
I wonder if it is possible to amortize allocations here somehow? Using Pools? Do we really to return a mutable `List`?

Reply:
How often it is called?

Reply:
I find pools quite dangeours in this context, the architecture is not strong enough to hold them. I rather not risk somebody not returning `PlaceCardComponentModel` to the pool and having multiple systems modifying the same instance.